integration/qbft/tests/setup_test.go

```diff
@@ -75,10 +76,13 @@ func TestMain(m *testing.M) {
 		Nodes: nodes,
 	}
 
-	m.Run()
+	// exitCode required here only for linter
+	exitCode := m.Run()
 
 	//teardown
 	for i := 0; i < len(ln.Nodes); i++ {
 		_ = ln.Nodes[i].Close()
 	}
+
+	os.Exit(exitCode)
```

Comment:
can we use [require.Zero](https://pkg.go.dev/github.com/stretchr/testify/require#Zero)?

Reply:
@nkryuchkov, you can't use `require.Zero` as it needs `t *testing.T` and we have here `testing.M`